modules/travel_pay/app/services/travel_pay/authenticated_session.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class AuthenticatedSession < TravelPay::Session
```

Comment:
This seems to be a duplicate (or nearly so) of the auth_manager. I vote keep auth_manager (the naming makes more sense to my poor addled brain 😁).

Reply:
I swear I deleted this... Thanks for catching